### ISSUE TITLE
How to remove a member from the Where of an MDXBuilder

### ISSUE THREAD

**Issue by RobbyMeyers** [cubewise-code/mdxpy#50]
Is there a way to remove a member from the where part of the MDX builder?  It seems to be additive when I try to replace with something else.

**Comment by MariusWirtz** [comment]
You are right @RobbyMeyers. We are missing a `remove_member_from_where` function on the `MdxBuilder` class.
I suggest we leave this issue open and I tag it as an enhancement.


For the moment, you can use the below workaround.

``` python
from mdxpy import MdxBuilder, Member, MdxTuple, MdxHierarchySet

q = MdxBuilder.from_cube("Sales")
q.add_hierarchy_set_to_row_axis(MdxHierarchySet.tm1_subset_all("Product"))
q.add_hierarchy_set_to_column_axis(MdxHierarchySet.tm1_subset_all("Customer"))
q.where(Member.of("Year", "2025"), Member.of("Version", "Actual"))
print(q.to_mdx())

q._where = MdxTuple([m for m in q._where.members if m.dimension != "Year"])
print(q.to_mdx())
```

**Comment by MariusWirtz** [comment]
solved with #52
